website/cue/reference/remap/functions/set_semantic_meaning.cue

```diff
@@ -0,0 +1,44 @@
+package metadata
+
+remap: functions: set_semantic_meaning: {
+	category: "Event"
+	description: """
+		Sets a semantic meaning for an event. Note that this function assigns
+		meaning at Vector startup, and has _no_ runtime behavior. It is suggested
+		to put all calls to this function at the beginning of a VRL function. The function
+		cannot be conditionally called (eg: using an if statement cannot stop the meaning
+		from being assigned).
+		"""
+
+	arguments: [
+		{
+			name: "target"
+			description: """
+				The path of the value that will be assigned a meaning.
```

Comment:
This is probably a question on the sink side of things, but if the path resolves to an object or an array... the sink will probably error on that?

Reply:
The requirement also requires a specific type, so it that doesn't match, it will error. Generally strings, ints, or timestamps are required.